gnd/src/main/java/com/google/android/gnd/ui/home/featuresheet/RecordListViewModel.java

```diff
@@ -28,20 +29,54 @@
 import com.google.android.gnd.vo.Record;
 import java.util.Collections;
 import java.util.List;
+
+import io.reactivex.Flowable;
+import io.reactivex.processors.BehaviorProcessor;
 import java8.util.Optional;
 import java8.util.stream.Collectors;
 import javax.inject.Inject;
 
 // TODO: Roll up into parent viewmodel. Simplify VMs overall.
 public class RecordListViewModel extends AbstractViewModel {
+
+  class ArgumentWrapper {
```

Comment:
I've been putting inner classes at the end of the class instead of the beginning.

Also, is there a more natural name for this? `ArgumentWrapper` and `argumentProcessor` don't tell us anything about what they contain and when they're used.